src/constants/externalPages.ts

```diff
@@ -0,0 +1,3 @@
+export enum ExternalPages {
+  WAGENT_API = 'https://api.wagent.app',
```

Comment:
this is not an external page. Also, take the WAGENT_API from .env too (update .env.example also)